Out of date documentation (#12407)
aspnet/AspNetCore.Docs: aspnetcore/client-side/using-grunt.md

Grunt is configured using a manifest named *Gruntfile.js* that defines, loads and registers tasks that can be run manually or configured to run automatically based on events in Visual Studio.

> 1. Right-click the project and select **Add > New Item**. Select the **Javascript file** option, change the name to *Gruntfile.js*, and click the **Add** button.

Reviewer: ```suggestion
1. Right-click the project and select **Add** > **New Item**. Select the **JavaScript File** item template, change the name to *Gruntfile.js*, and click the **Add** button.
```

> (this is no longer correct, as you get an empty javascript file.  assuming the code is still correct, it should probably say:)

Reviewer: ```suggestion
```

>    Add the following code to the Javascript file. The `initConfig()` is used to set options for each package, and the remainder of the module will load and register tasks.

Reviewer: ```suggestion
   Add the following code to *Gruntfile.js*. The `initConfig` function sets options for each package, and the remainder of the module loads and register tasks.
```
